src/auth/downscopedclient.ts

```diff
@@ -129,39 +148,41 @@ export class DownscopedClient extends AuthClient {
 
   /**
    * Provides a mechanism to inject Downscoped access tokens directly.
-   * When the provided credential expires, a new credential, using the
-   * external account options are retrieved.
-   * Notice DownscopedClient is the broker class mainly used for generate
-   * downscoped access tokens, it is unlikely we call this function in real
-   * use case.
-   * We implement to make this a helper function for testing all cases in getAccessToken().
+   * The expiry_date field is required for other clients dealing with
+   * token expiration checking and refreshing.
```

Comment:
The expiry_date field is required to facilitate determination of the token expiration which would make it easier for the token consumer to handle.